devicetypes/smartthings/aeotec-child-siren.src/aeotec-child-siren.groovy

```diff
@@ -0,0 +1,66 @@
+/**
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition(name: "Aeotec Child Siren", namespace: "smartthings", author: "SmartThings") {
+        capability "Actuator"
+        capability "Health Check"
+        capability "Alarm"
```

Comment:
If you add capability "Configuration", then `configure()` would be called automatically